<?xml version="1.0"?>

<launch>

<arg name="pose_topic" 	default="mavros/vision_pose/pose" /> 
<arg name="setpoint_topic" 	default="mavros/setpoint_position/local" />
<arg name="hover_alt" 	default="2.0" /> 
<arg name="acceptance_rad" 	default="0.2" /> 

<node pkg="waypoint_navigation" name="waypoint_node" type="waypoint_node.py" output="screen"> 
	<param name="pose_topic" 		value="$(arg pose_topic)"/> 
	<param name="setpoint_topic" 	value="$(arg setpoint_topic)"/> 
	<param name="hover_alt" 		value="$(arg hover_alt)"/> 
	<param name="acceptance_rad" 	value="$(arg acceptance_rad)"/> 
</node>


</launch>